config/initializers/timeout.rb

```diff
@@ -0,0 +1,2 @@
+# Temporarily increasing to account for HybridBill document uploads
+Rails.application.config.middleware.insert_before Rack::Runtime, Rack::Timeout, service_timeout: 5
```

Comment:
can we make this disablable with a `ENV` flag? Something like:
```ruby
Rails.application.config.middleware.insert_before Rack::Runtime, Rack::Timeout, service_timeout: 5 unless ENV['DISABLE_TIMEOUT']
```

Then usage would be:
```bash
DISABLE_TIMEOUT=true bundle exec rails s
```

If we do this, please also include details in the README